models/Definition.php

```diff
@@ -171,30 +189,42 @@ protected function makeUrlSet()
         }
 
         $xml = $this->makeXmlObject();
+        $xss = $xml->createProcessingInstruction('xml-stylesheet',
+            'type="text/xsl" href="' . Url::asset('/plugins/winter/sitemap/assets/sitemap.xsl') . '"'
+        );
+        $xml->appendChild($xss);
+
         $urlSet = $xml->createElement('urlset');
         $urlSet->setAttribute('xmlns', 'http://www.sitemaps.org/schemas/sitemap/0.9');
+        $urlSet->setAttribute('xmlns:xhtml', 'http://www.w3.org/1999/xhtml');
         $urlSet->setAttribute('xmlns:xsi', 'http://www.w3.org/2001/XMLSchema-instance');
         $urlSet->setAttribute('xsi:schemaLocation', 'http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd');
 
+        /**
+         * @event winter.sitemap.makeUrlSet
+         * Provides an opportunity to interact with a sitemap UrlSet after its creation
+         *
+         * Example usage:
+         *
+         *   Event::listen('winter.sitemap.makeUrlSet', function ((Definition) $definition, (DomDocument) $xml, (ElementNode) $urlSet) {
+         *       $urlSet->setAttribute('xmlns:xhtml', 'http://www.w3.org/1999/xhtml');
+         *   });
+         *
+         */
+        Event::fire('winter.sitemap.makeUrlSet', [$this, $xml, $urlSet]);
```

Comment:
Do we still need this event?

Reply:
You managed to make it look less usefull (grin), but I'd like to keep it as it makes the plugin more extensible.

Reply:
Can you think of uses that it could be used for? It could always be added later 😉

Reply:
It's not hurting, and it's only triggered once per page request.

Reply:
Ah yes, death by a thousand paper cuts :) I don't like adding new events in general (especially when there isn't a hard need for it) because while it does make things more extendable, it adds irrevocably to the public API and incurs a performance / maintenance penalty no matter how slight. Additionally, just because you have an event present doesn't mean that it's the right event in the right place for any potential future extensibility needs, so the value of said event being added before there's a reason for it to be used is on even shakier ground IMO. 

Any thoughts with regards to those points?

Reply:
Those are all valid points. I'll remove the event.